app.js

```diff
@@ -11,10 +11,7 @@ const port = process.env.NODE_ENV === 'development' ? '3001' : '3000';
 
 // database
 app.locals.pool = createPoolCluster();
-app.locals.pool.add('local', config.mysql0);
-app.locals.pool.add('db1', config.mysql1);
-app.locals.pool.add('db2', config.mysql2);
-app.locals.pool.add('db3', config.mysql3);
+config.forEach((db, dsn) => app.locals.pool.add(db, dsn));
```

Comment:
should use more descriptive variable names. 

dsn here would be the index. I don't think that's what you meant to pass to `pool.add`. Think you want to do 
```
config.forEach((dbConfig, index) => app.locals.pool.add(`db${index}`, dbConfig));

```

Reply:
it doesn't seem to work that way, it seems to be index in db. I tested this.

Reply:
dsn = database source name or basically the db config itself

Reply:
should be used as pool.add("name of connection", "config obj")

Reply:
looked at docs, dont understand why this works as is.

Reply:
ok tried it with 

config.forEach((dsn, i) => app.locals.pool.add(i, dsn));

and it doesn't work.

Reply:
configArry is an array populated with an object. 

```
configArry = [{
     user: process.env.DB_USER || 'root',
     password: process.env.DB_PASSWORD || 'root',
     database: process.env.NODE_ENV === 'development' ? 'khajana_dev_khajana' : 'khajana_khajana',	
     dateStrings: true,	 
     acquireTimeout: 5000,
     connectionLimit: process.env.DB_POOL_SIZE,
     host: db.host,
     port: db.port || 3306,
}];
```

looping over it with foreach takes a [callback](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Description) with params (value, index). 

Are your sure this is working?

Reply:
do 

```
config.forEach((dsn, i) => app.locals.pool.add(`db${i}`, dsn));
```